.github/workflows/ci_test-conda.yml

```
@@ -19,7 +19,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.8"]  # previous to last Python version as that one is already used in test-full
-        pytorch-version: ["1.7", "1.8", "1.9", "1.10"]  # nightly: add when there's a release candidate
+        pytorch-version: ["1.8", "1.9", "1.10"]  # nightly: add when there's a release candidate
```

```suggestion
        pytorch-version: ["1.7", "1.8", "1.9", "1.10"]  # nightly: add when there's a release candidate
```

Rolling back some commits to see what went wrong.